### PR TITLE
fastq-screen: new package

### PR DESCRIPTION
**Pull request by akthoma**
adding new package fastq-screen and perl dependencies

### CODE REVIEW

File: var/spack/repos/builtin/packages/fastq-screen/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class FastqScreen(Package):
+    """FIXME: Put a proper description of your package here."""
```
Comment (davydden) [inline var/spack/repos/builtin/packages/fastq-screen/package.py]: fixme

File: var/spack/repos/builtin/packages/perl-extutils-pkgconfig/package.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
```
Comment (junghans) [inline var/spack/repos/builtin/packages/perl-extutils-pkgconfig/package.py]: 2017?

**Comment by akthoma** [comment]
@junghans Do you suggest I do a different branch/pr for each package? I don't think any of the pull requests I make I will ever want partially merged--if my packages are together in a branch it means that they are dependencies and the main package will not work if partially merged.

**Comment by akthoma** [comment]
Closed and reopened to restart travis-ci.

**Pull request merged**